### PR TITLE
add global random seed

### PR DESCRIPTION
**Pull request by nirtiac**
Works on GPU and CPU, independently..

### CODE REVIEW

**Comment by nirtiac** [comment]
Also works when combining with the new speedup pull request, provided that num_workers is not set nor is the cudnn backend optimizer.

File: vulcanai/__init__.py [hunk 1]
```diff
@@ -14,3 +20,19 @@
                           defaults={'logfilename': log_output_path})
 
 logger = logging.getLogger(__name__)
+
+
+def random_seeding(seed_value):
```
Comment (rfratila) [inline vulcanai/__init__.py]: should we call it something like `set_global_seed()`?

Comment (nirtiac) [reply vulcanai/__init__.py]: sure, that's better!

File: vulcanai/__init__.py [hunk 2]
```diff
@@ -14,3 +20,19 @@
                           defaults={'logfilename': log_output_path})
 
 logger = logging.getLogger(__name__)
+
+
+def set_global_seed(seed_value):
+    """
+    Sets all the random seeds.
+    :param seed_value: seed value
```
Comment (rfratila) [inline vulcanai/__init__.py]: Can you update this to the more verbose way we do function docs please?

Comment (nirtiac) [reply vulcanai/__init__.py]: yeah, sorry, bad IDE defaults. done.

**Comment by rfratila** [comment]
It looks good to me so once the tests failures are resolved we’re good to go.

**Comment by nirtiac** [comment]
Just removed tabular dataset's random seed method... will also integrate into the new tabulardataset fix and test conclusively there.

**Comment by rfratila** [comment]
good catch. I don't believe there should be other seed references - but I would double check. I look forward to learn more about your test results!

**Comment by nirtiac** [comment]
note: this conflicts with other current pull requests and needs to be merged last to preserve all tabulardataset changes.

**Pull request merged**